ext/ftp/php_ftp.c

```diff
@@ -1596,6 +1615,8 @@ PHP_FUNCTION(ftp_get_option)
 		RETURN_FALSE;
 	}
 
+	char returnbuffer[18];
+	unsigned char *bytes = (unsigned char *) &ftp->externalip.s_addr;
```

Comment:
These two variables has to be declared at the top of the block, however since they are used in the switch statement below, the case can be wrapped in {} for those two

PHP targets C89, hence the top level decls